### PR TITLE
Update README.rst

### PR DESCRIPTION
**Pull request by roberttomas**
I added a cd ./NCAR-pangeo-tutorial/  in step 2), after the instructions to log out and log back in to make changes take effect.

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -37,8 +37,13 @@ If you are unsure about any setting, accept the defaults. We recommend adding th
...
 .. NOTE::
 
-  To make the changes take effect, logout and log back in.
+To make the changes take effect, logout and log back in.
+  
+Change into the newly created NCAR-pangeo-tutorial directory
 
+.. code:: bash 
+
+   cd ./NCAR-pangeo-tutorial/
```
Comment (matt-long) [inline README.rst]: I think this should read 

```
cd /path/to/NCAR-pangeo-tutorial
```

**Comment by roberttomas** [comment]
That sounds good.

What are the next steps?

File: README.rst (hunk 1 above)
Comment (roberttomas) [reply README.rst]: That sounds good

**Comment by matt-long** [comment]
@roberttomas, please make the change and commit to your fork.

**Pull request merged**